compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/checkers/FirHelpers.kt

```diff
@@ -320,3 +325,10 @@ internal fun throwableClassLikeType(session: FirSession) = session.builtinTypes.
 
 fun ConeKotlinType.isSubtypeOfThrowable(session: FirSession) =
     throwableClassLikeType(session).isSupertypeOf(session.typeContext, this.fullyExpandedType(session))
+
+// TODO: better to set [FirValueParameter]'s isVal and isVar so it can be read from FIR element directly
```

Comment:
You can remove this TODO.

Reply:
And a bunch of unused imports as well (e.g., VAL_KEYWORD)